README.md

```diff
@@ -249,6 +249,10 @@ To Add yourself as a Definition Owner:
 
 Once a week the Definition Owners are synced to the file [.github/CODEOWNERS](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/.github/CODEOWNERS) which is our source of truth.
 
+#### tslint.json
+
+Shouldn't have any additional rules. Just leave the content as `{ "extends": "dtslint/dt.json" }`. If for some reason some rule needs to be disabled, [disable it for that specific line](https://palantir.github.io/tslint/usage/rule-flags/#comment-flags-in-source-code:~:text=%2F%2F%20tslint%3Adisable%2Dnext%2Dline%3Arule1%20rule2%20rule3...%20%2D%20Disables%20the%20listed%20rules%20for%20the%20next%20line) using `// tslint:disable-next-line:[ruleName]` — not for the whole package, so that disabling can be reviewed.
+
 #### Common mistakes
```

Comment:
Since linting is closer to the verification, I think that it would be best to move the "Common mistakes" so it's after verification.  (So the order is linting -- verification -- common mistakes.)

Should be updated in the TOC too.

Reply:
Done :heavy_check_mark: